src/wallet.js

```diff
@@ -0,0 +1,90 @@
+import { existsSync, readFileSync, writeFileSync } from 'fs'; // Import the fs module for file operations
+import { GetMempool } from './Mempool';
+import { Transaction, UnsignedTransaction } from './Transaction';
+
+
+class Wallet {
+    constructor() {
+        if (existsSync('private_key.json')) {
+            const { PrivateKey, Password } = this.loadFromFile();
+            this.PrivateKey = PrivateKey;
+            this.Password = Password;
+        } else {
+            this.Password = this.generatePassword();
+            this.PrivateKey = this.generatePrivatePemString(this.password);
+            this.saveToFile();
+        }
+        this.PublicKey = this.generatePublicPemString(this.PrivateKey, this.Password);
+    }
+
+    sendMoney(ReceiverPks, Msgs) {
+        let MoneyToSend = 0;
+        for (const m of Msgs) {
+            MoneyToSend += m;
+        }
+        const Tx = this.createTransaction(this.getUTXOs(MoneyToSend), ReceiverPks, Msgs);

```

Comment:
We need to validate the result received after creating a transaction. What if generation fails?